build/Targets/Settings.props

```diff
@@ -71,6 +71,9 @@
 
     <RoslynNetSdkRootPath>$(MSBuildSDKsPath)\Microsoft.NET.Sdk\Sdk\</RoslynNetSdkRootPath> 
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
+
+    <ProjectAssetsFile Condition="'$(BuildingForLiveUnitTesting)' != 'true'">$(BaseIntermediateOutputPath)project.assets.json</ProjectAssetsFile>
+    <ProjectAssetsFile Condition="'$(BuildingForLiveUnitTesting)' == 'true'">$(LiveUnitTestingOriginalBaseIntermediateOutputPath)project.assets.json</ProjectAssetsFile>
```

Comment:
So, I did some more investigation, and it appears the reason this isn't working is that the Roslyn projects don't do ```<Project Sdk="Microsoft.NET.Sdk" ...```  The explanation is somewhat complicated, but this is the only way that LUT will currently work for projects with ```<ProjectReference>```.  Until we either think of a clever way to work around this limitation in LUT or we change the Roslyn projects to use the Sdk attribute, I'd still like to make this change to enable dogfooding of LUT.